### ISSUE TITLE
Future session ideas

### ISSUE THREAD

**Issue by frnsys** [sunday-school/cybernetics-club#18]
Dumping some possibilities for future sessions...if we get enough maybe we can be deliberate about ordering them?
- Chaos theory/complexity science
- Organizational cybernetics/design
- Economics + cybernetics (e.g. steady state economics, _Economics Becomes a Cyborg Science_)

**Comment by parkan** [comment]
interested in option 3 in particular

**Comment by dantaeyoung** [comment]
All sound great! I'm particularly interested in the second one.

Another topic (which could very well be incorporated into any of these) could be 'Applied Cybernetics', or 'actionable heuristics', or 'How To Incorporate Cybernetics Into Your Own Life' -- as in cybernetics. 

I'm not quite sure the specifics of what this would entail, but I'm thinking of a set of discussions around what it means to 'use' cybernetics in a personal life. Since it seems like military/organizational/business has utilized cybernetic principles early on, it might be interesting to find other applicable versions. 

(Is 'being cybernetic' about being mindful of undersea fiber cables when you watch Netflix? Not taking Uber, or taking Uber? In the same way that there's an 'ethics' of conscious food consumption (artisanal, free-range, vegetarian-fed, organic, etc?), is there an ethics of cybernetics-y infrastructure awareness? Does one incorporate a
https://en.wikipedia.org/wiki/OODA_loop into one's own life to self-diagnose one's own strategy? Etc. etc.)

**Comment by frnsys** [comment]
yes! i love that idea @provolot. maybe there are two threads there: 1) "thinking cybernetically" more broadly (e.g. undersea fiber cables, food production, etc) but maybe also 2) "cybernetic introspection", thinking about yourself, your own thought processes, etc, cybernetically (not sure if this is part of what you meant though)

**Comment by frnsys** [comment]
also i hadn't heard of the OODA loop before but there's a pentagram in there lol:

![](https://upload.wikimedia.org/wikipedia/commons/thumb/3/3a/OODA.Boyd.svg/929px-OODA.Boyd.svg.png)

**Comment by tarngerine** [comment]
All sound dope. Introspection and economy 

On Wed, Sep 28, 2016 at 1:37 AM -0400, "Francis Tseng" notifications@github.com wrote:

also i hadn't heard of the OODA loop before but there's a pentagram in there lol:

https://upload.wikimedia.org/wikipedia/commons/thumb/3/3a/OODA.Boyd.svg/929px-OODA.Boyd.svg.png

**Comment by urcades** [comment]
Such an amazing diagram 💯  

The idea of a 'Self-Applied Cybernetics' would be a fun thread to follow, if only to begin formulating methods of self-regulation as an individual within a greater and more complex world.

After reading through Inventing the future, it got me thinking about how using cybernetics as a lens thru which to visualize the political landscape would be cool! Within the book, there was a brief mention of CyberSyn that highlighted an attempt at regulating an entire socialist country via cybernetic methods, which was really cool to read about within the context of the entire book.

I can see points 1 & 2 being a nice meld of sorts, which could be explored in and of themselves in multiple sessions. Perhaps we could structure the next 3 or so sessions as an intentional 'series' of sorts? idk!!!!

**Comment by frnsys** [comment]
also interesting:

design + cybernetics, across scales:

> cybernetics in urban planning > architecture/built environment > products or the parallel of software and its stack (- @edouerd)

**Comment by dantaeyoung** [comment]
Yes! Worth looking at re: urban planning/arch + cybernetics:

the architect cedric price and 'cybernetician' gordon pask:
http://www.candidejournal.net/article/the-mcappy-project/
http://www.bcchang.com/transfer/articles/2/18346584.pdf

**Comment by frnsys** [comment]
dropping "logistics" as a future topic

**Comment by dantaeyoung** [comment]
**Potential Topics:**
- Cybernetics & Us
- Organizational Cybernetics
- Economics & Cybernetics (Cybersyn, Red Plenty)
- Politics & Governance (Seeing like a State, anarchism and cybernetics)
- Cybernetics of The Alt-Right????? 
 (https://en.wikipedia.org/wiki/Cybernetic_Culture_Research_Unit)
Steve Bannon & Biosphere 2? (organized by @edouerd & @frnsys ?)
- Art & Cybernetics (organized by @melaniehoff & ?)
- Urban planning & Architecture & Cybernetics (Gordon Pask, Cedric Price, Local Code, Benjamin Bratton)
- Cybernetics & Spirituality
- Internet & Cybernetics?
- Logistics & Infrastructure & Cybernetics (https://en.wikipedia.org/wiki/Black_Star_Line, selections from The Stack / The Black Stack from e-flux, The Red Stack by Tiziana Terranova, Extrastatecraft)
- Destroying Cybernetic systems (Thomas Kuhn SSR, tipping point,  https://en.wikipedia.org/wiki/Excession)
- Critiques of Cybernetics (Anti-Cybernetics, more Sick Systems, military origin, etc)
- Cybernetics & AI conference 

**PSAs**:
- Kira is guest-hosting Fei's Bel-air.org show TRYTOBEGOOD : on voting rights & law on Monday, Feb 20th, 8pm
- Learning Gardens - #mondays channel - on Meanings: at Monday Feb 27th,  7pm (location NEW INC)
- Dan's starting a The Stack Reading Group - Saturday, March 11, 2-4pm (location TBD).. Slack group here: https://spatial-stack-auto-invite.herokuapp.com/

**Communication Channels?**
- Email vs Slack vs Github
- Emails for meeting readings, calendar invites?
- Slack for discussions?
- Let's try integrations!

**Comment by wileycount** [comment]
I imagine it's further along in the future, but I'd be glad to (co)organize the cybernetics + architecture/planning. 

It might be interesting to simply frame it as "Cybernetics and Space" in order to capture the range of possible spatial and temporal scales and to encapsulate the complexity of the system.

**Comment by frnsys** [comment]
@wileycount definitely feel free to get started on that one - "Cybernetics and Space" sounds great. you can open a separate issue and start soliciting/dumping readings into there :)
